packages/salesforcedx-vscode-core/src/commands/baseDeployRetrieve.ts

```diff
@@ -217,6 +325,16 @@ export abstract class RetrieveExecutor<T> extends DeployRetrieveExecutor<T> {
     return operation.pollStatus();
   }
 
+  protected async handleSourceConflictError(e: any) {
+    // Retrieve operation - proceed and do not handle or throw.
```

Comment:
can this also go up above?

Reply:
👍🏽 Updated this method!  LMK what you think of the updates.